### ISSUE TITLE
Highlight the selected event in week view all day events section

### ISSUE THREAD

**Issue by severus1990** [twinssbc/Ionic2-Calendar#26]
Is there a way to customize the calendar? In the week view where the full day events are shown, the height of event is set as 25px and also the top is calculated as a multiple of 25px. Is there a way to alter these?

I have a requirement to change the color of the clicked event so that the user will know which event is clicked. Is that possible? I can see this happening in month view

Thanks in advance.

**Comment by twinssbc** [comment]
@severus1990 Adding view customisation is on my list, has got time to implement it.

**Comment by twinssbc** [comment]
@severus1990 I have added view customization in version 0.1.0. You can get more information from README. Feel free to have a try. Thanks.

**Comment by severus1990** [comment]
@twinssbc I tried using the customization in my existing project. I am getting the below error:
polyfills.js:3 Unhandled Promise rejection: Template parse errors:
Can't bind to 'weekviewNormalEventTemplate' since it isn't a known property of 'calendar'

Am I missing something? I tried an **npm install ionic2-calendar --save**  hoping to update the calendar version. Still I am getting the same error.
